src/commands/trigger.cpp

```diff
@@ -0,0 +1,728 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// trigger.cpp
+//
+// Identification: src/commands/trigger.cpp
```

Comment:
Do we really want to name this directory "commands"? That doesn't seem obvious. Maybe it should be called "trigger".